contracts/BallotsStorage.sol

```diff
@@ -0,0 +1,109 @@
+pragma solidity ^0.4.24;
+
+import "./abstracts/VotingEnums.sol";
+import "./eternal-storage/EternalStorage.sol";
+import "./ProxyStorage.sol";
+import "./Consensus.sol";
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+contract BallotsStorage is EternalStorage, VotingEnums {
+  using SafeMath for uint256;
+
+  event ThresholdChanged(uint256 indexed thresholdType, uint256 newValue);
+
+  modifier onlyOwner() {
+    require(msg.sender == addressStorage[keccak256(abi.encodePacked("owner"))]);
+    _;
+  }
+
+  modifier onlyVotingToChange() {
+    bool isVotingToChangeBlockReward = msg.sender == getVotingToChangeBlockReward();
+    bool isVotingToChangeMinStake = msg.sender == getVotingToChangeMinStake();
+    bool isVotingToChangeMinThreshold = msg.sender == getVotingToChangeMinThreshold();
+    require(isVotingToChangeBlockReward || isVotingToChangeMinStake || isVotingToChangeMinThreshold);
+    _;
+  }
+
+  function initialize(uint256[] _thresholds) public onlyOwner {
+    require(!isInitialized());
+    require(_thresholds.length == uint256(ThresholdTypes.MinStake));
+    uint256 thresholdType = uint256(ThresholdTypes.Voters);
+    for (; thresholdType <= _thresholds.length; thresholdType++) {
+      uint256 thresholdValue = _thresholds[thresholdType - uint256(ThresholdTypes.Voters)];
+      if (!setThreshold(thresholdValue, thresholdType)) {
+        revert();
+      }
+    }
+    setInitialized(true);
+  }
+
+  function setInitialized(bool _value) internal {
+    boolStorage[keccak256(abi.encodePacked("isInitialized"))] = _value;
+  }
+
+  function isInitialized() public view returns(bool) {
+    return boolStorage[keccak256(abi.encodePacked("isInitialized"))];
+  }
+
+  function getBallotLimitPerValidator() public view returns(uint256) {
```

Comment:
would love to see some code documentation, I don't think that I understand this indicator.

Reply:
This function returns the number of "open" (active) ballots each validator (someone with voting rights) can have at the same time.

Reply:
And what it supposed return value?

Reply:
There's a default value of 100 max open ballots at a time (`getMaxLimitBallot` function).
If there are validators, this function will return 100/number_of_validators.
We can decide we don't need any limits and remove this functionality totally.